docs/src/main/asciidoc/infinispan-client.adoc

```diff
@@ -109,15 +109,42 @@ quarkus.infinispan-client.cache.magazine.configuration=<distributed-cache><encod
 If both `configuration-uri` and `configuration` are configured for the same cache with the same Quarkus profile,
 `configuration-uri` gets preference over `configuration`.
 
+[TIP]
+====
+Cache configuration can be provided in XML, JSON or YAML. Use the Infinispan Console and the cache configuration Wizard
+to learn more about Infinispan Caches and create guided configurations.
```

Comment:
It would be awesome in the future if this where possible using the DevUI

Reply:
From the Dev UI we have a link to the console, I don't know if it's possible to bring the console framed into the dev ui

Reply:
It should be. @phillip-kruger?

Reply:
Yes definitely possible.  In the (soon to be) new Dev UI you can use the external link and that should happen automatically.

Reply:
That's awesome!